src/containers/notification/components/TwitterShare.tsx

```diff
@@ -149,6 +150,12 @@ export const getRemixCosignText = async (
     link
   }
 }
+
+export const getRewardsText = (notification: ChallengeReward) => ({
+  text: `I earned $AUDIO for completing challenges on @AudiusProject #AudioRewards`,
+  link: ''
```

Comment:
link?

Reply:
Maybe make the link optional and not require in the `openTwitterLink` func
ref: https://github.com/AudiusProject/audius-client/blob/be456a65c14b2a2c6e85b734c2b258f0f122eb37/src/utils/tweet.ts#L1

Reply:
Not sure if there is supposed to be a link for this, could be the profile page I guess? This is placeholder while I check with Julian